README.md

```diff
@@ -23,6 +24,7 @@ docker run yukinying/chrome-headless
 ### Node App
 ```bash
 docker pull quay.io/ukhomeofficedigital/html-pdf-converter
+docker build -t html-pdf-converter .
```

Comment:
I don't think this makes sense. You either want to run the remote image that you've pulled *or* build a local image from code, but not both.

What are you trying to do here?

Reply:
I couldn't get it working before without building the image.  Since I've upgraded my docker it's now all good so I shall close this PR